src/SourceBuild/tarball/content/README.md

```diff
@@ -0,0 +1,165 @@
+
+In the VMR, you can find:
+- source files of [each product repository](#list-of-components) which are mirrored inside of their respective directories under [`src/`](src/),
+- tooling that enables [building the whole .NET product from source](https://github.com/dotnet/source-build) on Linux platforms,
+- small customizations, in the form of patches, applied on top of the original code to make the build possible,
+- *[in future]* E2E tests for the whole .NET product.
+
+More in-depth documentation about the VMR can be found in [VMR Design And Operation](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Design-And-Operation.md#layout).  
+See also [dotnet/source-build](https://github.com/dotnet/source-build) for more information about our whole-product source-build.
+
+## Goals
+
+The main purpose of this repository is to have all source code necessary to build the whole .NET product from source contained in any given commit.
+
+Further, the VMR eventually aims to become the place from which we release and service future versions of .NET.
+This is to reduce the complexity of the product construction process and thus enable partners and 3rd parties to easily build, test and modify .NET using their custom infrastructure.
+
+Furthermore, we hope to solve other problems that the current multi-repo setup brings:
+- Enable the standard [down-/up-stream open-source model](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Upstream-Downstream.md).
+- Fulfill requirements of .NET distro builders such as RedHat or Canonical to natively include .NET in their distribution repositories.
+- Simplify scenarios such as client-run testing of bug fixes and improvements. The build should work in an offline environment too for certain platforms.
+- Enable developers to make and test changes spanning multiple repositories.
+- More efficient pipeline for security fixes during the CVE pre-disclosure process.
+
+## Limitations
+
+**This is a work-in-progress and an experiment.**
+There are considerable limitations to what is possible at the moment. For an extensive list of current limitations, please see [Temporary Mechanics](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Design-And-Operation.md#temporary-mechanics).
+
+The VMR is expected to become non-experimental by .NET 8 Preview 1 (Februrary, 2023) 
+This means it won't be short-lived anymore and we won't be reserving the right to delete and re-create it anymore.
+Other limitations might apply until the .NET 9 timeframe.
+See the [Unified Build roadmap](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild) for more details.
+
+### Supported platforms / builds
+
+.NET Source-Build is supported on the oldest available .NET SDK feature update for each major release, and on Linux only.
```

Comment:
We should drop the oldest available .NET SDK part all together.  The only part that is relevant is that .NET source build only supported Linux.

Reply:
Removed. Additionally, I moved the "we will have main and release branches" above under the "in the VMR you can find" section.

[`a0aaa40` (#14848)](https://github.com/dotnet/installer/pull/14848/commits/a0aaa409b850090520c9c889e3c8c1da4a4ae9b5)